rpcd/etc/openstack_deploy/conf.d/ceph.yml.aio

```diff
@@ -25,5 +25,3 @@ storage_hosts:
           rados_connect_timeout: -1
           glance_api_version: 2
           volume_backend_name: ceph
-          rbd_user: "{{ cinder_ceph_client }}"
```

Comment:
Actually, I prefer to remove them since they are essentially cruft at this point.